std/os/index.zig

```diff
@@ -116,7 +116,7 @@ pub fn getRandomBytes(buf: []u8) !void {
                 else => return unexpectedErrorPosix(errno),
             }
         },
-        Os.macosx, Os.ios => return getRandomBytesDevURandom(buf),
+        Os.macosx, Os.ios, Os.freebsd => return getRandomBytesDevURandom(buf),
```

Comment:
note: FreeBSD 12 has getrandom, I used it on purpose

Reply:
Why not use getrandom?

Reply:
because getrandom just landed on FreeBSD 12 and I was considering we were going to support version 11, but as @myfreeweb  said if it's ok to no support, we can bring it back

Reply:
I see, thanks.

> I think it's fine to not support versions older than 12 in a young language.

I agree with this statement.